src/layout/header/Header.tsx

```diff
@@ -59,20 +66,28 @@ export const Header = () => {
 
   const isTicketCurator = hasCuratorRole(user);
 
-  const notificationsQuery = useQuery({
+  const dialogueNotificationsParams = getDialogueNotificationsParams(user?.nvaUsername);
+  const dialogueNotificationsQuery = useQuery({
+    enabled: !!user?.isCreator && !!dialogueNotificationsParams.owner,
+    queryKey: ['notifications', dialogueNotificationsParams],
```

Comment:
Kunne det vært lurt å lagt på `'dialogue'` i key'en her for å lettere skille mellom denne og `taskNotificationsQuery`? 
```suggestion
    queryKey: ['notifications', 'dialogue', dialogueNotificationsParams],
```

Reply:
kanskje bare gi dem navn `dialogueNotifications` og `taskNotifications` er like greit 🤔

Reply:
Ser nå at `queryKey: ['taskNotifications']` burde holde også, at vi ikke trenger den ektra biten med params her, siden dette er statiske ting uansett

Reply:
nei, nvm, da klager linting, så tar med den